app/screens/vaccination/AddReminder.tsx

```diff
@@ -670,7 +670,7 @@ useEffect(() => {
                           editReminderItem ? new Date(measureTimeDefined) : new Date()
                         }
                         mode={'time'}
-                        display="default"
+                       display="spinner"
```

Comment:
Is this only change you made or am I missing something? Did we not replace the calendar library in json file?

Reply:
No there is nothing missed, We didn't change existing calendar library.